src/perlDebug.ts

```diff
@@ -715,6 +720,15 @@ class PerlDebugSession extends LoggingDebugSession {
 	 * Stacktrace
 	 */
 	private async stackTraceRequestAsync(response: DebugProtocol.StackTraceResponse, args: DebugProtocol.StackTraceArguments): Promise<DebugProtocol.StackTraceResponse> {
+
+		// FIXME(bh): There is probably a better way to re-use the code
+		// in that function that does not require setting up a malformed
+		// object here, but this seems good enough for the moment.
+		await this.loadedSourcesRequestAsync(
+			{} as DebugProtocol.LoadedSourcesResponse,
+			{}
+		);
```

Comment:
we could extract the code to reuse into its own method

Reply:
I do not see a good way to do so at the moment.